[2020-04-08 02:06:38,538] INFO Terminating process due to signal SIGHUP (kafka.Kafka$)
[2020-04-08 02:06:38,541] INFO [KafkaServer id=0] shutting down (kafka.server.KafkaServer)
[2020-04-08 02:06:38,542] INFO [KafkaServer id=0] Starting controlled shutdown (kafka.server.KafkaServer)
[2020-04-08 02:06:38,561] INFO [KafkaServer id=0] Controlled shutdown succeeded (kafka.server.KafkaServer)
[2020-04-08 02:06:38,564] INFO [SocketServer brokerId=0] Shutting down (kafka.network.SocketServer)
[2020-04-08 02:06:38,570] INFO [SocketServer brokerId=0] Shutdown completed (kafka.network.SocketServer)
[2020-04-08 02:06:38,570] INFO [Kafka Request Handler on Broker 0], shutting down (kafka.server.KafkaRequestHandlerPool)
[2020-04-08 02:06:38,582] INFO [Kafka Request Handler on Broker 0], shut down completely (kafka.server.KafkaRequestHandlerPool)
[2020-04-08 02:06:38,585] INFO [ThrottledRequestReaper-Fetch]: Shutting down (kafka.server.ClientQuotaManager$ThrottledRequestReaper)
[2020-04-08 02:06:38,594] INFO [GroupCoordinator 0]: Member kafka-python-1.4.7-b4421065-7a8f-4bbb-9732-59b49d0e1cd0 in group my-group has failed, removing it from the group (kafka.coordinator.group.GroupCoordinator)
[2020-04-08 02:06:38,594] INFO [GroupCoordinator 0]: Preparing to rebalance group my-group with old generation 2 (__consumer_offsets-12) (kafka.coordinator.group.GroupCoordinator)
[2020-04-08 02:06:38,595] INFO [GroupCoordinator 0]: Group my-group with generation 3 is now empty (__consumer_offsets-12) (kafka.coordinator.group.GroupCoordinator)
[2020-04-08 02:06:38,798] INFO [ThrottledRequestReaper-Fetch]: Stopped (kafka.server.ClientQuotaManager$ThrottledRequestReaper)
[2020-04-08 02:06:38,798] INFO [ThrottledRequestReaper-Fetch]: Shutdown completed (kafka.server.ClientQuotaManager$ThrottledRequestReaper)
[2020-04-08 02:06:38,799] INFO [ThrottledRequestReaper-Produce]: Shutting down (kafka.server.ClientQuotaManager$ThrottledRequestReaper)
[2020-04-08 02:06:39,798] INFO [ThrottledRequestReaper-Produce]: Stopped (kafka.server.ClientQuotaManager$ThrottledRequestReaper)
[2020-04-08 02:06:39,798] INFO [ThrottledRequestReaper-Produce]: Shutdown completed (kafka.server.ClientQuotaManager$ThrottledRequestReaper)
[2020-04-08 02:06:39,798] INFO [ThrottledRequestReaper-Request]: Shutting down (kafka.server.ClientQuotaManager$ThrottledRequestReaper)
[2020-04-08 02:06:40,798] INFO [ThrottledRequestReaper-Request]: Stopped (kafka.server.ClientQuotaManager$ThrottledRequestReaper)
[2020-04-08 02:06:40,798] INFO [ThrottledRequestReaper-Request]: Shutdown completed (kafka.server.ClientQuotaManager$ThrottledRequestReaper)
[2020-04-08 02:06:40,799] INFO [KafkaApi-0] Shutdown complete. (kafka.server.KafkaApis)
[2020-04-08 02:06:40,801] INFO [ExpirationReaper-0-topic]: Shutting down (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2020-04-08 02:06:40,811] WARN Session 0x1715803d7500001 for server localhost/0:0:0:0:0:0:0:1:2181, unexpected error, closing socket connection and attempting reconnect (org.apache.zookeeper.ClientCnxn)
java.io.IOException: Connection reset by peer
	at java.base/sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at java.base/sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at java.base/sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:276)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:233)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:223)
	at java.base/sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:358)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:68)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:366)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)
[2020-04-08 02:06:40,912] INFO zookeeper state changed (Disconnected) (org.I0Itec.zkclient.ZkClient)
[2020-04-08 02:06:40,944] INFO [ExpirationReaper-0-topic]: Stopped (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2020-04-08 02:06:40,944] INFO [ExpirationReaper-0-topic]: Shutdown completed (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2020-04-08 02:06:40,955] INFO [TransactionCoordinator id=0] Shutting down. (kafka.coordinator.transaction.TransactionCoordinator)
[2020-04-08 02:06:40,956] INFO [ProducerId Manager 0]: Shutdown complete: last producerId assigned 0 (kafka.coordinator.transaction.ProducerIdManager)
[2020-04-08 02:06:40,957] INFO [Transaction State Manager 0]: Shutdown complete (kafka.coordinator.transaction.TransactionStateManager)
[2020-04-08 02:06:40,957] INFO [Transaction Marker Channel Manager 0]: Shutting down (kafka.coordinator.transaction.TransactionMarkerChannelManager)
[2020-04-08 02:06:40,958] INFO [Transaction Marker Channel Manager 0]: Stopped (kafka.coordinator.transaction.TransactionMarkerChannelManager)
[2020-04-08 02:06:40,958] INFO [Transaction Marker Channel Manager 0]: Shutdown completed (kafka.coordinator.transaction.TransactionMarkerChannelManager)
[2020-04-08 02:06:40,958] INFO [TransactionCoordinator id=0] Shutdown complete. (kafka.coordinator.transaction.TransactionCoordinator)
[2020-04-08 02:06:40,958] INFO [GroupCoordinator 0]: Shutting down. (kafka.coordinator.group.GroupCoordinator)
[2020-04-08 02:06:40,959] INFO [ExpirationReaper-0-Heartbeat]: Shutting down (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2020-04-08 02:06:41,022] INFO [ExpirationReaper-0-Heartbeat]: Stopped (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2020-04-08 02:06:41,022] INFO [ExpirationReaper-0-Heartbeat]: Shutdown completed (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2020-04-08 02:06:41,023] INFO [ExpirationReaper-0-Rebalance]: Shutting down (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2020-04-08 02:06:41,054] INFO [ExpirationReaper-0-Rebalance]: Stopped (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2020-04-08 02:06:41,054] INFO [ExpirationReaper-0-Rebalance]: Shutdown completed (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2020-04-08 02:06:41,055] INFO [GroupCoordinator 0]: Shutdown complete. (kafka.coordinator.group.GroupCoordinator)
[2020-04-08 02:06:41,057] INFO [ReplicaManager broker=0] Shutting down (kafka.server.ReplicaManager)
[2020-04-08 02:06:41,057] INFO [LogDirFailureHandler]: Shutting down (kafka.server.ReplicaManager$LogDirFailureHandler)
[2020-04-08 02:06:41,058] INFO [LogDirFailureHandler]: Shutdown completed (kafka.server.ReplicaManager$LogDirFailureHandler)
[2020-04-08 02:06:41,057] INFO [LogDirFailureHandler]: Stopped (kafka.server.ReplicaManager$LogDirFailureHandler)
[2020-04-08 02:06:41,058] INFO [ReplicaFetcherManager on broker 0] shutting down (kafka.server.ReplicaFetcherManager)
[2020-04-08 02:06:41,060] INFO [ReplicaFetcherManager on broker 0] shutdown completed (kafka.server.ReplicaFetcherManager)
[2020-04-08 02:06:41,061] INFO [ExpirationReaper-0-Fetch]: Shutting down (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2020-04-08 02:06:41,120] INFO [ExpirationReaper-0-Fetch]: Stopped (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2020-04-08 02:06:41,120] INFO [ExpirationReaper-0-Fetch]: Shutdown completed (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2020-04-08 02:06:41,121] INFO [ExpirationReaper-0-Produce]: Shutting down (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2020-04-08 02:06:41,143] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-04-08 02:06:41,146] WARN Session 0x1715803d7500001 for server null, unexpected error, closing socket connection and attempting reconnect (org.apache.zookeeper.ClientCnxn)
java.net.ConnectException: Connection refused
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)
[2020-04-08 02:06:41,271] INFO [ExpirationReaper-0-Produce]: Stopped (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2020-04-08 02:06:41,271] INFO [ExpirationReaper-0-Produce]: Shutdown completed (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2020-04-08 02:06:41,272] INFO [ExpirationReaper-0-DeleteRecords]: Shutting down (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2020-04-08 02:06:41,471] INFO [ExpirationReaper-0-DeleteRecords]: Stopped (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2020-04-08 02:06:41,471] INFO [ExpirationReaper-0-DeleteRecords]: Shutdown completed (kafka.server.DelayedOperationPurgatory$ExpiredOperationReaper)
[2020-04-08 02:06:41,475] INFO [ReplicaManager broker=0] Shut down completely (kafka.server.ReplicaManager)
[2020-04-08 02:06:41,476] INFO Shutting down. (kafka.log.LogManager)
[2020-04-08 02:06:41,754] INFO Shutdown complete. (kafka.log.LogManager)
[2020-04-08 02:06:41,759] INFO Terminate ZkClient event thread. (org.I0Itec.zkclient.ZkEventThread)
[2020-04-08 02:06:42,383] INFO Opening socket connection to server localhost/0:0:0:0:0:0:0:1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-04-08 02:06:42,485] INFO EventThread shut down for session: 0x1715803d7500001 (org.apache.zookeeper.ClientCnxn)
[2020-04-08 02:06:42,485] INFO Session: 0x1715803d7500001 closed (org.apache.zookeeper.ZooKeeper)
[2020-04-08 02:06:42,488] INFO [KafkaServer id=0] shut down completed (kafka.server.KafkaServer)
